pkgs/by-name/mu/mum/package.nix

```diff
@@ -0,0 +1,64 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  cmake,
+  pkg-config,
+  alsa-lib,
+  gdk-pixbuf,
+  glib,
+  libnotify,
+  libopus,
+  openssl,
+  versionCheckHook,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "mum";
+  version = "0.5.1";
+  src = fetchFromGitHub {
+    owner = "mum-rs";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-r2isuwXq79dOQQWB+CsofYCLQYu9VKm7kzoxw103YV4=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-ey3nT6vZ5YOZGk08HykK9RxI7li+Sz+sER3HioGSXP0=";
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+
+  buildInputs = [
+    alsa-lib
+    gdk-pixbuf
+    glib
+    libnotify
+    libopus
+    openssl
+  ];
+
+  postInstall = ''
+    install -Dm 755 documentation/mumctl.1 $out/share/man/man1/mumctl.1
+    install -Dm 755 documentation/mumd.1 $out/share/man/man1/mumd.1
+    install -Dm 755 documentation/mumdrc.5 $out/share/man/man5/mumdrc.5
+  '';
```

Comment:
use `installManPages` for this.

Reply:
No, there is a binary for installing man pages I believe.

Reply:
Got it!